lib/util/mixins.js

```diff
@@ -42,34 +40,33 @@ module.exports = {
 		return options;
 	},
 
+	getQueueName ( messageType, prefix ) {
```

Comment:
This doesn't take into account the fact that they use to be able to pass in `options.queue`. https://github.com/sinet/lapin/blob/6828afa1dd2ef2070b6a79ce24676a4febe03bc4/lib/util/mixins.js#L93-L94